dotnet/src/Plugins/Plugins.Web/WebSearchEnginePlugin.cs

```diff
@@ -55,9 +56,16 @@ public WebSearchEnginePlugin(IWebSearchEngineConnector connector)
         {
             throw new InvalidOperationException("Failed to get a response from the web search engine.");
         }
+        var hasNonAscii = results.Any(result => result.Any(c => c > 127));
 
+        var jso = new JsonSerializerOptions
+        {
+            Encoder = hasNonAscii
+                     ? JavaScriptEncoder.UnsafeRelaxedJsonEscaping
+                     : JavaScriptEncoder.Default
```

Comment:
If the presence of a non-ASCII character will cause this to use UnsafeRelaxedJsonEscaping, why not use it if it's only ASCII, too?